insights/specs/default.py

```diff
@@ -129,8 +126,8 @@ def tomcat_base(broker):
     dig_edns = simple_command("/usr/bin/dig +edns=0 . SOA")
     dig_noedns = simple_command("/usr/bin/dig +noedns . SOA")
     dirsrv = simple_file("/etc/sysconfig/dirsrv")
-    dirsrv_access = glob_file("var/log/dirsrv/*/access")
-    dirsrv_errors = glob_file("var/log/dirsrv/*/errors")
+    dirsrv_access = glob_file("/var/log/dirsrv/*/access")
+    dirsrv_errors = glob_file("/var/log/dirsrv/*/errors")
```

Comment:
This doesn't appear to match what is in the uploader.json, is the json correct?

Reply:
@bfahr - I'm not sure about why the uploader.json is updated as below:
```
"file": "/var/log/dirsrv/.*/()*(errors|errors\\.2.*)"
```
in https://github.com/RedHatInsights/insights-frontend-assets/commit/550bb7a

@wcmitchell, would you mind to take a look at the above commit? Why the spec is modified:
from `"/var/log/dirsrv/.*/()*errors"` to `"/var/log/dirsrv/.*/()*(errors|errors\\.2.*)"`?
I'm sorry that I can't find any corresponding information about this modification.

Reply:
@xiangce I discussed this with @kylape and this spec is broken in the uploader.json because the regex will not work for the directory segment.  I'm going to log an issue on that project and will go ahead and merge this PR.